modules/python/src2/cv2.cpp

```diff
@@ -1170,6 +1170,41 @@ PyObject* pyopencv_from(const int64& value)
     return PyLong_FromLongLong(value);
 }
 
+template<>
+bool pyopencv_to(PyObject* obj, int64& value, const ArgInfo& info)
```

Comment:
Bindings generator doesn't support `int64` type in functions signatures. No plans to add that due to ambiguous using and compatibility issues with other languages. Consider using `int` / `size_t` instead.

It makes sense to add note here.

Reply:
Can I implement `pyopencv_to(int64_t)` with static_cast to `size_t` ?

Reply:
Main problem with `size_t` is that it is unsigned.

Reply:
Removed this implementation. Looks like it isn't necessary to read `int64` data from python so far.